src/panels/logbook/ha-logbook.ts

```diff
@@ -253,21 +291,33 @@ export class HaLogbook extends LitElement {
       return;
     }
 
+    this._logbookEntries = [...newEntries].reverse();
+  }
+
+  private _appendNewEntries = (newEntries: LogbookEntry[]) => {
```

Comment:
These two function should be smart enough to append or prepend the rows depending on the times.  An optimization we can do is if they select a long time window we can select the end of the time window right away and then deliver the older results in a second message

Reply:
efe9393d1709bd625004d236aa7479f00e9cad01